tests/OpenTK.Tests.Integration/OpenTK.Tests.Integration.fsproj

```diff
@@ -0,0 +1,1678 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>522d9279-3ed6-475f-867a-6ae69a53c24a</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <RootNamespace>OpenTK.Tests.Integration</RootNamespace>
+    <AssemblyName>OpenTK.Tests.Integration</AssemblyName>
+    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFSharpCoreVersion>4.4.0.0</TargetFSharpCoreVersion>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <Name>OpenTK.Tests.Integration</Name>
+    <Deterministic>true</Deterministic>
+    <StartAction>Program</StartAction>
```

Comment:
They allow you to "Start debugging" the project in Visual Studio. Without them VS is like "How do you run a class library?"